src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -151,11 +151,9 @@ private void ingestConversationsPerUser() {
         if (this.globalProperties.getBoolean(PROPERTY_INGEST_MESSAGES)) {
             this.loginRestClient.loginBankAdmin();
             this.userContextPresentationRestClient.selectContextBasedOnMasterServiceAgreement();
-            this.accessControlSetup.getLegalEntitiesWithUsersExcludingSupport().stream()
-                .map(LegalEntityWithUsers::getUserExternalIds)
-                .flatMap(List::stream)
-                .collect(Collectors.toList())
-                .forEach(this.messagesConfigurator::ingestConversations);
+            this.accessControlSetup.getLegalEntitiesWithUsersExcludingSupport().forEach(legalEntityWithUsers ->
+                this.messagesConfigurator.ingestTopics()
```

Comment:
We do not use `legalEntityWithUsers` in the `this.messagesConfigurator.ingestTopics()`. Is that OK?

Reply:
Yes, that's okay because topics are set up by the admin. We only want to show you can set up different topics per LE